### DISCUSSION TITLE
blog/seeding-database-with-drizzle-orm/

### DISCUSSION THREAD

**Discussion by anasrar** [anasrar/anasrar.github.io#47] [Giscus]
---

**Comment by vioreed** [comment]
nice article

**Comment by deadcoder0904** [comment]
thanks for this. i was using typescript with import statements so it gave error.

removed `esbuild-register` & used [tsx](https://github.com/privatenumber/tsx?tab=readme-ov-file) to make it work.

seed script also simplified:

> "db:seed": "tsx ./src/seed/index.ts"

**Reply by anasrar** [reply]
Thanks for the feedback, gonna update this post next week. Can you provide more information like Node version and some snippet code, I can't quite understand with "typescript with import statements".

**Reply by deadcoder0904** [reply]
try using seed database with import statements using your current blog post to see the errors. i don't remember the complete error now but my solution fixes it.
